### DISCUSSION TITLE
Where can I download Laragon with Xdebug enabled and working?

### DISCUSSION THREAD

**Discussion by mawg** [leokhoa/laragon#731] [Q&A]
It's late, I'm tired, and I can't seem to get Xdebug working. 

I just need Apache, PHP & MySQL, but I can't seem to get Xdebug working out of the box. Is there something that I can download that will give me what I want?  I forgot to mention in the title that I need the portable version

**Comment by Pen-y-Fan** [comment]
The version of Xdebug is specific to the PHP version, so not easy to release in one version. I have a blog on [How to Set up VS Code to use PHP with Xdebug 3 on Windows](https://pen-y-fan.github.io/2021/08/03/How-to-Set-up-VS-Code-to-use-PHP-with-Xdebug-3-on-Windows/), which may help. There is a section on Xdebug in [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/#xdebug) too.
